src/rgw/rgw_iam_policy.h

```diff
@@ -215,7 +216,7 @@ struct ARN {
   ARN(const rgw_obj& o);
   ARN(const rgw_bucket& b);
   ARN(const rgw_bucket& b, const std::string& o);
-
+  ARN(const RGWGroup& g);
```

Comment:
@adamemerson : I have made changes related to Group ARN, obviating the need to store ARN as part of group info. Please review these changes.